scripts/SANS/sans/algorithm_detail/move_workspaces.py

```diff
@@ -9,7 +9,10 @@
 import math
 from mantid.api import MatrixWorkspace
 from abc import (ABCMeta, abstractmethod)
-from sans.state.StateObjects.StateMoveDetectors import StateMove
+
+from sans.state.AllStates import AllStates
+from sans.state.StateObjects.StateMoveDetectors import StateMove, StateMoveLOQ, StateMoveLARMOR, StateMoveZOOM, \
```

Comment:
Its good practise to put these in alphabetical order - very picky I know!

Reply:
No it's a fair shout, it's a shame my IDE doesn't do this whilst optimising imports :+1: